### DISCUSSION TITLE
Updated Proposal

### DISCUSSION THREAD

**Discussion by scummings-tech** [Hinrichsta/FA23-Cyber8420#13] [Proposal]
All, 

I will be placing my updated proposal I worked on today in our teams folder. In the meantime, please start working on the tasks we were assigned based upon our team meeting this week.

Have a great weekend!

**Comment by lreichlinger** [comment]
Hey All,

Just wanted to mention that I have uploaded the latest Diagram file called "Project-Proposal-Systems-Engineering-View-3". Someone mentioned the Database systems as being part of the system-of-interest or just a system element. I am happy to allow other to make changes or update as needed. Please let me know if you need anything from me.
